Description: Sort the package lists to improve the reproducibility
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/biz.aQute.bndlib/src/aQute/bnd/osgi/Packages.java
+++ b/biz.aQute.bndlib/src/aQute/bnd/osgi/Packages.java
@@ -1,9 +1,9 @@
 package aQute.bnd.osgi;
 
 import java.util.Collection;
-import java.util.LinkedHashMap;
 import java.util.Map;
 import java.util.Set;
+import java.util.TreeMap;
 
 import aQute.bnd.header.Attrs;
 import aQute.bnd.osgi.Descriptors.PackageRef;
@@ -21,11 +21,11 @@
 	}
 
 	public Packages(Packages other) {
-		map = new LinkedHashMap<>(other.map);
+		map = new TreeMap<>(other.map);
 	}
 
 	public Packages() {
-		map = new LinkedHashMap<>();
+		map = new TreeMap<>();
 	}
 
 	@Override
